tests/smoke.rs

```diff
@@ -913,15 +913,18 @@ fn cargo_check_check() {
 #[test]
 fn cargo_check_check_no_ctoml() {
     fn run() -> Result<()> {
+        const TARGET: &str = "i686-pc-windows-gnu";
         let project = HProject::new(false)?;
+        // should be OK as long as other tests don't use the same target
```

Comment:
Maybe add a comment that we need to do this because this is a cross-test using `HProject` instead of `Project`.

Reply:
done